default.nix

```diff
@@ -700,7 +602,7 @@ in let this = rec {
   # The systems that we want to build for on the current system
   cacheTargetSystems = [
     "x86_64-linux"
-    "i686-linux"
+    # "i686-linux"
```

Comment:
Is this intentional?

Reply:
Yes i686-linux is pretty broken in Nixpkgs right now. I believe it was GHC that wasn't building at all.